source/precalculus/source/08-TE/01.ptx

```diff
@@ -12,6 +12,1187 @@
+        <answer>
+          <p>
+            <m>\sin^4\beta=\left(\frac{1-\cos2\beta}{2}\right)^2</m>
+          </p>
+        </answer>
+      </task>
+
+      <task>
+        <statement>
+          <p>
+            Square the right-hand side of the equation you found in part (b).
+          </p>
+        </statement>
+        <answer>
+          <p>
+            <m>\sin^4\beta=\frac{1-2\cos2\beta+\cos^22\beta}{4}</m>
+          </p>
+        </answer>
+      </task>
+
+      <task>
+        <statement>
+          <p>
+            Notice that you still have a cosine function being squared in your equation in part (c). Substitute the <m>\cos^2\beta</m> using the power reduction formula in <xref ref="def-power-reducing-formulas"/> and then simplify
+          </p>
+        </statement>
+        <answer>
+          <p>
+            <m>\frac{2-4\cos\beta+1+\cos4\beta}{8}</m> or <m>\frac{3-4\cos2\beta+\cos4\beta}{8}</m>
+          </p>
+        </answer> 
+      </task>
+    </activity>
+
+    <activity xml:id="deriving-half-angle-formulas">
+      <introduction>
+        <p>
+          The next set of identities is the set of half-angle formulas, which can be derived from the reduction formulas. Let's derive the half-angle formula for <m>\sin\frac{\theta}{2}</m>.
+        </p>
+      </introduction>
+
+      <task>
+        <statement>
+          <p>
+            Recall the power reduction formula: <m>\sin^2\theta=\frac{1-\cos2\theta}{2}</m>. Replace <m>\theta</m> with <m>\frac{\theta}{2}</m>.
+          </p>
+        </statement>
+        <answer>
+          <p>
+            <m>\sin^2\left(\frac{\theta}{2}\right)=\frac{1-\cos2\left(\frac{\theta}{2}\right)}{2}</m>
+          </p>
+        </answer>
+      </task>
+
+      <task>
+        <statement>
+          <p>
+            Simplify the right-hand side of your equation in part (a).
+          </p>
+        </statement>
+        <answer>
+          <p>
+            <m>\sin^2\left(\frac{\theta}{2}\right)=\frac{1-\cos\theta}{2}</m>
+          </p>
+        </answer>
+      </task>
+
+      <task>
+        <statement>
+          <p>
+            Now take the square root of both sides to solve for <m>\sin\theta</m>.
+          </p>
+        </statement>
+        <answer>
+          <p>
+            <m>\sin\left(\frac{\theta}{2}\right)=\pm\sqrt{\frac{1-\cos\theta}{2}}</m>
+          </p>
+        </answer>
+      </task>
+    </activity>
+
+    <remark>
+      The derivation for <m>\cos\left(\frac{\theta}{2}\right)</m> and <m>\tan\left(\frac{\theta}{2}\right)</m> is similar to that in <xref ref="deriving-half-angle-formulas"/> when starting with the power-reduction formulas for each trigonometric function.
+    </remark>
+
+    <definition xml:id="def-half-angle-formulas">
+      <statement>
+        <p>
+          The <term>Half Angle Formulas</term> are:
+          <ul>
+            <li>
+              <p>
+                <m>\sin\frac{\alpha}{2}=\pm\sqrt{\frac{1-\cos\alpha}{2}}</m>
```

Comment:
I don't disagree but I feel like most other texts list the formulas with both. Is it better with the note?

Reply:
It's better, but it's still not correct.  It suggests there are two values for a particular theta.  Compare this to the way we use $\pm$ in the quadratic formula, where it is important that there really are two values there.

Also, just because other texts do something does not bind us to do it that way.  We can be better and not write incorrect things.

Reply:
I believe I fixed the issues we had on this but it's been a while since we talked about it. Can you double check what I did is good now?

Reply:
I think [Theorem 8.1.26](https://3e56e8a3.tbil.pages.dev/preview/precalculus/instructor/TE1#thm-half-angle-formulas) still needs to be updated to match what is in the [appendix]https://tbil.org/preview/precalculus/instructor/list-of-trig-identities.html#identity-half-angle)

Reply:
It could perhaps also do with a fluency builder at the end that makes use of one of the negative versions (I feel like we discussed that at some point but not sure it got documented or implemented anywhere).